access-token-management/src/AccessTokenManagement.OpenIdConnect/OpenIdConnectTokenManagementServiceCollectionExtensions.cs

```diff
@@ -130,6 +150,28 @@ public static IHttpClientBuilder AddUserAccessTokenHttpClient(this IServiceColle
             .AddUserAccessTokenHandler(parameters);
     }
 
+    /// <summary>
+    /// Adds a typed HTTP client for the factory that automatically sends the current client access token. The client access token is an access token that is not associated with any user, obtained with the client credentials flow.
```

Comment:
The xmldoc description of the previously existing overload of this function (that takes a name parameter instead of a type parameter) also needs to be updated. Please copy this line to that overload, changing "typed HTTP client" to "named HTTP client".